.github/workflows/consuming.yml

```diff
@@ -31,6 +32,15 @@ jobs:
           - project: submariner
             cabledriver: wireguard
             deploytool: operator
+            k8s_version: 1.17.17
```

Comment:
```suggestion
```

Seems redundant?

Reply:
Here's a PR to demo the failure that results from removing this:

https://github.com/dfarrell07/shipyard/pull/14/checks?check_run_id=2185768163

Reply:
Ah, because if it’s *not* specified, we don’t provide a default, so the kind default is used instead (1.20.2).

Reply:
Yeah, the strange bit is that we do provide defaults that seem to work in every case except this one. 

If the new E2E GHA feature isn't used at all, the default works:

https://github.com/submariner-io/submariner/runs/2189218309?check_suite_focus=true

That should be caused by the default here:

https://github.com/submariner-io/shipyard/blob/devel/gh-actions/e2e/action.yaml#L7

If no k8s_version flag is passed to clusters.sh (skipping the GHA), it also handles the default correctly:

https://github.com/submariner-io/shipyard/pull/496/checks?check_run_id=2187434544

That should be caused by the default here:

https://github.com/submariner-io/shipyard/blob/devel/scripts/shared/clusters.sh#L6

Reply:
I think it's something weird about how GHA matrix includes work in this case. The default in the GHA is not being used which results in [setting `--k8s_version=""`](https://github.com/submariner-io/shipyard/blob/devel/gh-actions/e2e/action.yaml#L45), which I guess overrides the default in clusters.sh with null.

Anyway, can we move ahead? :sweat_smile: